libs/platform/src/lib/button/button.component.ts

```diff
@@ -43,13 +43,18 @@ export class ButtonComponent extends BaseComponent implements AfterViewInit {
     @Input()
     buttonType: ButtonType = 'standard';
 
+    /** Whether button is in toggled state. */
+    @Input()
+    toggled: Nullable<boolean>;
+
     /** arialabel, tooltip for truncated text
      * for acccesiblity of the element */
     @Input()
     title: Nullable<string>;
 
-    /** aria-selected for acccesiblity to
-     *  the native HTML button*/
+    /** @deprecated use toggled input property instead
+     * aria-selected for acccesiblity to the native HTML button
+     */
```

Comment:
Please add documentation entry like it is for Content Density, so that user sees it right when they arrive at the docs page